src/Fields/field_tuples.jl

```diff
@@ -1,62 +1,64 @@
 """
-    VelocityFields(arch, grid)
+    VelocityFields(arch, grid; u=zeros(arch, grid), v=zeros(arch, grid), w=zeros(arch, grid))
```

Comment:
I think this could be useful to users who want to partially-initialize `velocities` with a predefined field for one of `u, v w`, so I'm all for it.